tests/components/hue/test_bridge.py

```diff
@@ -192,6 +192,41 @@ async def test_hue_activate_scene(hass, mock_api):
     assert mock_api.mock_requests[2]["path"] == "groups/group_1/action"
 
 
+async def test_hue_activate_scene_transition(hass, mock_api):
+    """Test successful hue_activate_scene with transition."""
+    config_entry = config_entries.ConfigEntry(
+        1,
+        hue.DOMAIN,
+        "Mock Title",
+        {"host": "mock-host", "username": "mock-username"},
+        "test",
+        config_entries.CONN_CLASS_LOCAL_POLL,
+        system_options={},
+        options={CONF_ALLOW_HUE_GROUPS: True, CONF_ALLOW_UNREACHABLE: False},
+    )
+    hue_bridge = bridge.HueBridge(hass, config_entry)
```

Comment:
Don't create the bridge directly in the test. Set up the integration while patching out the library and let the integration create the bridge.

https://developers.home-assistant.io/docs/development_testing#writing-tests-for-integrations